.travis.yml

```diff
@@ -44,9 +44,6 @@ matrix:
           compiler: gcc
           env: GCC_VERSION=6
 
-        - os: osx
```

Comment:
I'm not sure to understand what's going on. Were we building the macOS version with clang twice?